mycli/sqlexecute.py

```diff
@@ -221,3 +224,16 @@ def server_type(self):
 
         self._server_type = (product_type, version)
         return self._server_type
+
+    def get_connection_id(self):
+        if not self.connection_id:
+            self.reset_connection_id()
+        return self.connection_id
+
+    def reset_connection_id(self):
+        # Remember current connection id
+        _logger.debug('Get current connection id')
+        res = self.run('select connection_id()')
+        for title, cur, headers, status in res:
+            self.connection_id = list(cur)[0][0]
```

Comment:
Instead of converting it to a list, can you use `fetchone()` to fetch the first row of the results and then index into the first column? Something like: 

```
self.connection_id = cur.fetchone()[0]
```